src/pages/BankTransfer/Index.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import "bootstrap/dist/css/bootstrap.min.css";
+import ToastSoS from "../../Components/Toast/Index";
+import ReceiverBankAccount from "../../Components/ReceiverBankAccount/Index";
+import NumberFormat from "react-number-format";
+
+function BankTransfer() {
```

Comment:
I think this should be called BankTransferPage to be consistent (the filename too)